### PR TITLE
completed task 'stars'

### PR DESCRIPTION
**Pull request by TKuts**
- [DEMO LINK](https://TKuts.github.io/layout_stars/)
- [TEST REPORT LINK](https://TKuts.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  margin-right: 4px;
+  background-position: 50%;
+}
+
+.stars--0 :nth-child(-n + 5),
+.stars--1 :nth-child(n + 2),
+.stars--2 :nth-child(n + 3),
+.stars--3 :nth-child(n + 4),
+.stars--4 :nth-child(n + 5) {
+  background-image: url("./images/star.svg");
```
Comment (lerastarynets) [inline src/style.css:26]: why not moving this style under `stars_star `selector?

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars :last-child {
+  margin: 0;
+}
```
Comment (lerastarynets) [inline src/style.css:11]: you can use your `margin-left: 4px;` with `:not(:first-child)` instead of the above

**Review by lerastarynets — CHANGES_REQUESTED** [review]
I see you did a hard work) Let's improve it a little bit

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  background-image: url("./images/star.svg");
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background-image: url("./images/star-active.svg");
+}
```
Comment (VitaliyBodnar) [inline src/style.css:34]: don't see any sense in adding a no-active background modificator, you can add a no-active start for every start
after that add modificator only for the active start
for instance 

Screenshot 2023-08-15 at 20 13 11

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
Almost done!

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
.

**Review by mykhalenych — APPROVED** [review]

**Pull request closed without merging**